sample/src/main/res/layout/activity_demo.xml

```diff
@@ -17,6 +17,7 @@
         android:text="@string/list_item_1"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_margin="10dp"
```

Comment:
Stick to 4dp grid for spacing and margins according to Material design, so 8dp or 12dp would be more appropriate here.